### PR TITLE
Entity type configuration

### PR DESCRIPTION
**Pull request by Abdurrasheed199**

### CODE REVIEW

File: src/Infrastructure/Persistence/EntityTypeConfigurations/ChandaTypeEntityTypeConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Persistence.EntityTypeConfigurations
+{
+    public class ChandaTypeEntityTypeConfiguration : IEntityTypeConfiguration<ChandaType>
+    {
+        public void Configure(EntityTypeBuilder<ChandaType> builder)
+        {
+            builder.ToTable("payments");
```
Comment (usman1607) [inline src/Infrastructure/Persistence/EntityTypeConfigurations/ChandaTypeEntityTypeConfiguration.cs]: change the table name to chanda_type

File: src/Infrastructure/Persistence/EntityTypeConfigurations/ChandaTypeEntityTypeConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            builder.Property(ct => ct.Code)
+                   .HasColumnName("code")
+                   .HasColumnType("varchar(255)")
+                   .IsRequired();
+
+            builder.HasIndex(ct => ct.Code)
+                   .IsUnique();
+
+            builder.Property(ct => ct.Description)
+                   .HasColumnName("description")
```
Comment (usman1607) [inline src/Infrastructure/Persistence/EntityTypeConfigurations/ChandaTypeEntityTypeConfiguration.cs]: Don't make description to be required, it is optional.

File: src/Infrastructure/Persistence/EntityTypeConfigurations/PaymentEntityTypeConfiguration.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        {
+            builder.ToTable("payments");
+
+            builder.HasKey(p => p.Id);
+
+            builder.Property(p => p.InvoiceId)
+                   .HasColumnName("invoice_id")
+                   .IsRequired();
+                
+            builder.HasIndex(p => p.InvoiceId)
```
Comment (usman1607) [inline src/Infrastructure/Persistence/EntityTypeConfigurations/PaymentEntityTypeConfiguration.cs]: InvoiceId cannot be unique, more than one payment can have the same InvoiceId

File: src/Infrastructure/Persistence/EntityTypeConfigurations/ChandaTypeEntityTypeConfiguration.cs (hunk 1 above)
Comment (Abdurrasheed199) [reply src/Infrastructure/Persistence/EntityTypeConfigurations/ChandaTypeEntityTypeConfiguration.cs]: Noted for action sir

**Pull request merged**